source/popup/components/ArchivesListPage.js

```diff
@@ -188,7 +188,7 @@ const Message = styled.div`
 function getProviderImage(archiveSourceType) {
     const imageSrc = ARCHIVE_IMAGES[archiveSourceType];
     if (!imageSrc) {
-        throw new Error(`No image asset for archive type: ${archiveSourceType}`);
+        return null;
```

Comment:
This is failing due to MyBcup archives being in my state. This change is not required in future versions, but just a monkey patch for now.